src/index.tsx

```diff
@@ -1,22 +1,22 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import reportWebVitals from './reportWebVitals';
 import { Provider } from 'react-redux';
+import reportWebVitals from './reportWebVitals';
 
 import App from './App';
 
 import './index.css';
-import { _store } from './redux';
+import { store } from './redux';
 
 const root = ReactDOM.createRoot(
-  document.getElementById('root') as HTMLElement
+  document.getElementById('root') as HTMLElement,
 );
 root.render(
   <React.StrictMode>
-    <Provider store={_store}>
+    <Provider store={store}>
```

Comment:
fair, I had some issues with making the attribute have the same name as the value, but if it works it works

Reply:
underscore suggests the value is unused so it is a listing error